backend/lcfs/web/api/compliance_report/services.py

```diff
@@ -182,19 +185,26 @@ async def create_analyst_adjustment_report(
         # Create the history record for the new supplemental report
         await self.repo.add_compliance_report_history(new_report, user)
 
+        # Copy over FSE
+        await self.final_supply_equipment_service.copy_to_report(
+            existing_report_id,
+            new_report.compliance_report_id,
+            current_report.organization_id,
+        )
+
         return ComplianceReportBaseSchema.model_validate(new_report)
 
     @service_handler
     async def create_supplemental_report(
-        self, report_id: int, user: UserProfile = None, legacy_id: int = None
+        self, original_report_id: int, user: UserProfile = None, legacy_id: int = None
```

Comment:
why the switch to original_report_id? this id can be any report in the chain, might add a bit of confusion

Reply:
I wanted a way to differentiate between the two reports, and report_id does not do that.